### DISCUSSION TITLE
Update docker image from Ubuntu 20.04 (focal) to 22.04 (jammy) or 24.04 (noble)

### DISCUSSION THREAD

**Discussion by candrews** [renovatebot/renovate#30789] [Suggest an Idea]
### Tell us more.

The Renovate docker images, such as [renovate/renovate](https://hub.docker.com/r/renovate/renovate) are currently based on Ubuntu 20.04 (focal).

Ubuntu 20.04 will have standard support until April 2025: https://ubuntu.com/about/release-cycle By that time, Renovate must update the version of Ubuntu being used to continue receiving security updates.

I think updating well before that time would also be beneficial.

**Comment by viceice** [comment]
- https://github.com/containerbase/base/pull/1991

currently blocked by mend
